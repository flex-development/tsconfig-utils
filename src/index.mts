/**
 * @file Package Entry Point
 * @module tsconfig-utils
 */

export type * from '#interfaces/index'
export * from '#lib/index'
export type * from '#types/index'
export type * from '@flex-development/tsconfig-types'
