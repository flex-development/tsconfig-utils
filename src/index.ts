/**
 * @file Package Entry Point
 * @module tsconfig-utils
 */

export * from '@flex-development/tsconfig-types'
export * from './interfaces'
export * from './utils'
