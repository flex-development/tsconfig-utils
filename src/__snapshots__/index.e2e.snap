// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`e2e:tsconfig-utils > should expose public api 1`] = `
[
  "isResolvedTsconfig",
  "isTsconfigHost",
  "loadTsconfig",
  "mergeTsconfig",
  "readTsconfig",
  "resolvePath",
]
`;
